en/phased-ranking.html

```diff
@@ -126,3 +126,180 @@ <h2 id="choosing-phased-ranking-functions">Choosing phased ranking functions</h2
 this to ensure that relevance is not hurt too much
 by not evaluating the real ranking function on all the hits.
 </p>
+
+
+
+<h2 id="rank-phase-statistics">Rank phase statistics</h2>
+<p>
+  Use <a href="reference/schema-reference.html#match-features">match-features</a>
+  and <a href="reference/schema-reference.html#summary-features">summary-features</a>
+  to export detailed match- and rank-information per query.
+  This requires post-processing and aggregation is an external system for analysis.
+</p>
+<p>
+  To evaluate how well the document corpus matches the queries,
+  use <a href="reference/schema-reference.html#mutate">mutable attributes</a>
+  to track how often each document survives each match- and ranking-phase.
+  This is aggregated per document and makes it easy to analyse using the query and grouping APIs in Vespa -
+  and no other processing/storage is required.
+</p>
+<p>
+  A mutable attribute is a number where an operation can be executed in 4 phases:
+</p>
+<ol>
+  <li>on-match</li>
+  <li>on-first-phase</li>
+  <li>on-second-phase</li>
+  <li>on-summary</li>
+</ol>
+<p>
+  The common use case is to increase the value by 1 for each execution.
+  With this, it is easy to evaluate the document's performance to the queries,
+  e.g. find the documents that appear in most queries, or the ones that never matched -
+  run a query and order by the mutable attribute.
+</p>
+{% include note.html content='The mutable attributes are just counters and memory-operations only -
+the values do not survive content node restarts. The values cannot be compared across nodes.
```

Comment:
Note: They might survive restarts if the mutable attribute was flushed to disk in the meantime, but this is something you cannot rely on.